pkg/scheduler/core/equivalence/eqivalence_test.go

```diff
@@ -190,31 +180,23 @@ func TestRunPredicate(t *testing.T) {
 		{
 			name:             "pod fits/cache hit",
 			pred:             mockPredicate{},
-			cache:            &upToDateCache{},
+			cache:            &schedulertesting.FakeCache{},
 			expectFit:        true,
 			expectCacheHit:   true,
 			expectCacheWrite: false,
 		},
 		{
 			name:             "pod fits/cache miss",
 			pred:             mockPredicate{fit: true},
-			cache:            &upToDateCache{},
+			cache:            &schedulertesting.FakeCache{},
 			expectFit:        true,
 			expectCacheHit:   false,
 			expectCacheWrite: true,
 		},
-		{
-			name:             "pod fits/cache miss/no write",
```

Comment:
Is there a new test that covers this scenario?

Reply:
This test is kept and updated now.
Found that `schedulercache.Cache` can be replaced by `generation` field.